src/components/DetailsKeyShortcutsWrapper/DetailsKeyShortcutsWrapper.test.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import { createMemoryHistory } from 'history';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+
+import {
+  renderWithReduxForm,
+  translationsProperties,
+} from '../../../test/jest/helpers';
+
+import {
+  duplicateRecordShortcut,
+  openEditShortcut,
+} from '../../../test/jest/helpers/shortcuts';
+
+import { OCLC_UPDATE_INSTANCE_JOB_ID } from '../../utils';
+
+import { DetailsKeyShortcutsWrapper } from './DetailsKeyShortcutsWrapper';
```

Comment:
Add `export * from './shortcuts';` to the index file in `helpers` folder, please

```suggestion
import React from 'react';
import { createMemoryHistory } from 'history';

import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';

import '../../../test/jest/__mock__';
import {
  renderWithReduxForm,
  translationsProperties,
  duplicateRecordShortcut,
  openEditShortcut,
} from '../../../test/jest/helpers';

import { DetailsKeyShortcutsWrapper } from './DetailsKeyShortcutsWrapper';

import { OCLC_UPDATE_INSTANCE_JOB_ID } from '../../utils';
```